rpcs/blockService_test.go

```diff
@@ -492,7 +492,7 @@ func addBlock(t *testing.T, ledger *data.Ledger) (timestamp int64) {
 	blk, err := ledger.Block(ledger.LastRound())
 	require.NoError(t, err)
 	blk.BlockHeader.Round++
-	blk.BlockHeader.TimeStamp += int64(crypto.RandUint64() % 100 * 1000)
+	blk.BlockHeader.TimeStamp += int64(crypto.RandUint64() % 100000 * 1000)
```

Comment:
This is perfectly fine, but just pointing out, because I think it's nice, that Go let's you use `100_000_000` so that it reads easier.

Reply:
note, % takes precedence over * - see https://go.dev/ref/spec#Operator%20precedence

Reply:
ok, ok, you'e right. But `100_000` also reads quicker than `100000` to me.